src/snkit/network.py

```diff
@@ -603,3 +604,43 @@ def set_precision(geom, precision):
     geom_mapping = mapping(geom)
     geom_mapping['coordinates'] = np.round(np.array(geom_mapping['coordinates']), precision)
     return shape(geom_mapping)
+
+
+def to_networkx(network):
+    """Return a networkx graph
+    """
+    # init graph
+    G = nx.Graph()
+    # get nodes from network data
+    G.add_nodes_from(network.nodes.id.to_list())
+    # get edges from network data
+    edges_as_list = [(network.edges.loc[i].from_id, 
+                      network.edges.loc[i].to_id) 
+                      for i in network.edges.index]
```

Comment:
This might be clearer using `network.edges.itertuples()` or `zip(network.edges.from_id, network.edges.to_id)`. Is there much difference in speed either way?

Reply:
Just checked the speeds on my sample. List comprehension was much slower (1.35 seconds) than `itertuples()` (0.01 seconds) and `zip` (0.001 seconds). I will change it to the `zip` method. Thanks for the tip!